SPECS/distribution-gpg-keys/distribution-gpg-keys.spec

```diff
@@ -1,7 +1,9 @@
 Summary:	GPG keys of various Linux distributions
 Name:		distribution-gpg-keys
-Version:	1.104
+Version:	1.105
 Release:	1%{?dist}
+Vendor:     Microsoft Corporation
+Distribution: Azure Linux
 License:	CC0
 URL:		https://github.com/rpm-software-management/distribution-gpg-keys
 # Sources can be obtained by
```

Comment:
This looks a bit ugly. We try to hold a higher bar for core packages, where the source tarball is taken directly from an upstream or we have a `generate_source_tarball.sh` script for hand-made tarballs. Script's name is important, as it's ran by our auto-upgrade pipelines.

Reply:
Good call-out. The comment is erroneous, inherited from the Fedora version of this spec. I've removed the comment to reflect that it's really just taken from the listed upstream source.